### ISSUE TITLE
When the second terminal is open, the commands are still sent to the first

### ISSUE THREAD

**Issue by arditshala** [miguelmota/terminal-tab#4]
I am trying to use this library and its open() method, so my goal is to open N terminals and run different command on them, however when I am doing something like the following:

```
const terminalTab = require('terminal-tab');
terminalTab.open('echo hello; echo hello2');
terminalTab.open('echo hello3; echo hello4');
```

Both of the terminal windows are being open successfully, however on the first one I get all the 4 echos, the second terminal is just empty. 

Is this library (method) suppose to work with more than one terminal?

**Comment by miguelmota** [comment]
@arditshala what terminal are you using? trying to reproduce

**Comment by arditshala** [comment]
@miguelmota Default MacBook Pro Terminal (MacOS 10.15.2)

**Comment by miguelmota** [comment]
It looks like a delay is required between open calls to prevent it from opening in same window. 
Should be fixed now in version `0.0.8`

**Comment by miguelmota** [comment]
Closed by https://github.com/miguelmota/terminal-tab/commit/b4b20e8770e1926256ea025ef7ab8885451d21dd
